latex/Inhalt/design.tex

```diff
@@ -63,6 +63,7 @@ \subsection{Interrupts}
 \end{minted}
 
 Wenn Interrupts erlaubt sind, wird der übergebene Opcode ausgeführt und der Boolean geflipped, andernfalls wird eine Errormeldung zurückgegeben. \rust{interrupts_enabled} ist initial \rust{false} und muss durch die entsprechende Instruktion (\asm{EI}) gesetzt werden.
+Prinzipiell kann also jede 1 Byte große Instruktion (parameterlose Instruktion) als Interrupt ausgeführt werden, meistens wird jedoch eine der \asm{RST}-Instruktionen ausgeführt. Diese sind \asm{CALL}-Instruktionen, die zu einer fixen Speicheradresse springen. An diesen fixen Adressen kann der Programmierer die entsprechenden Interrupt-Routinen platzieren.
```

Comment:
I think you should remove the "also", to me it sounds like implying knowledge for the reader, that might not be present at that point in time (which is nothing horrible but sounds kinda odd if you ask me).